optuna/samplers/_cmaes.py

```diff
@@ -413,6 +426,24 @@ def _get_trials(self, study: "optuna.Study") -> List[FrozenTrial]:
         return complete_trials
 
 
+def _split_optimizer_str(optimizer_str: str) -> Dict[str, str]:
+    optimizer_len = len(optimizer_str)
+    attrs = {}
+    for i in range(math.ceil(optimizer_len / _SYSTEM_ATTR_MAX_LENGTH)):
+        start = i * _SYSTEM_ATTR_MAX_LENGTH
+        end = min((i + 1) * _SYSTEM_ATTR_MAX_LENGTH, optimizer_len)
+        attrs["cma:optimizer:{}".format(i)] = optimizer_str[start:end]
+    return attrs
+
+
+def _concat_optimizer_attrs(optimizer_attrs: Dict[str, str]) -> str:
+    optimizer_str = ""
+    for i in range(len(optimizer_attrs)):
+        key = "cma:optimizer:{}".format(i)
+        optimizer_str += optimizer_attrs[key]
```

Comment:
I would generally use `str.join` on a list or with a list comprehension for repeated string concatenations (for performance), but I guess it doesn't make a lot of difference for just 2 or 3 iterations. Just a comment.

Reply:
Thank you for your comment.
I pushed https://github.com/optuna/optuna/pull/1833/commits/62c3996f01ee7acf80c3c1a42e2d6c96d8dc8b36.